### ISSUE TITLE
New Membership Acitivities Default

### ISSUE THREAD

**Issue by jkotin** [Princeton-CDH/mep-django#764]
Currently, when a visitor clicks on "activities" on a membership page, the visitor sees the member's "borrowing activity" first:

https://shakespeareandco.princeton.edu/members/abbott-berenice/borrowing/

Because most members do not have extant lending library cards, visitors usually see the message, "No documented borrowing activity."  I worry that the message discourages some visitors from exploring further.

Suggested change: when a visitor clicks on "activities" on the membership page, the visitor sees the member's "membership activity" first:

https://shakespeareandco.princeton.edu/members/abbott-berenice/membership/

In other words, membership activity should be the default page, not borrowing activity. 

An alternative, but more complicated solution: membership activity is the default only if there is no borrowing activity.
